addons/mail/static/src/models/follower.js

```diff
@@ -176,6 +192,10 @@ registerModel({
         followedThread: one('Thread', {
             inverse: 'followers',
         }),
+        followedThreadAsCurrentPartnerFollower: one('Thread', {
```

Comment:
`followedThreadAsCurrentPartnerFollowing`?

Reply:
or maybe not, keep like now just for consistency

Reply:
I have chosen `followedThreadAsFollowerOfCurrentPartner` so that the part after "as" is exactly the same as the inverse field. IMO `AsFollowerOfCurrentPartner` it's clearer to understand than `AsCurrentPartnerFollowing`.

Reply:
Changes done